packages/db/src/schema/monitors/constants.ts

```diff
@@ -33,7 +33,7 @@ export const monitorPeriodicity = [
   "1h",
   "other",
 ] as const;
-export const monitorMethods = ["GET", "POST", "HEAD"] as const;
+export const monitorMethods = ["GET", "POST", "HEAD", "PUT"] as const;
```

Comment:
we don't allow PUT here I think we would need to do a DB migration let's remove it now and let's fix it later with #619